### DISCUSSION TITLE
variables from another scene

### DISCUSSION THREAD

**Discussion by AlexKiryanov** [friction2d/friction#399] [Q&A] [answered]
Hello everyone! Hi @rodlie
Whether i can use variables from other scenes?
I need to take the properties of one object that is in another scene.

my_var x = Scene02.CC_object.transform.translation.x;
He doesn’t understand that expression.

**Comment by rodlie** [comment]
You can't reference other scenes in expressions.

This is of course something we can look at in a future release (v1.1+).

**Comment by AlexKiryanov** [comment, marked as answer]
Good day, again! Yes it would be a very useful function.
Thank you so much!
